src/gui/editors/PianoRoll.cpp

```diff
@@ -788,9 +775,19 @@ QColor PianoRoll::markedSemitoneColor() const
 void PianoRoll::setMarkedSemitoneColor( const QColor & c )
 { m_markedSemitoneColor = c; }
 
-void PianoRoll::drawNoteRect(QPainter & p, int x, int y, 
+float PianoRoll::noteOpacity() const
+{ return m_noteOpacity; }
+
+void PianoRoll::setNoteOpacity( const float & f )
```

Comment:
>Could we maybe use note color to denote velocity?
... ? We already use the color to denote velocity, the lightness of the color gets changed. The thing is, having two identical notes on top of each other would probably sound just as loud as a note with a double velocity, and if you need to know the exact velocity, you have the velocity bars on the bottom.